script.js

```diff
@@ -0,0 +1,117 @@
+// Speakers page
+const speakers = [
+
+  {
+    name: 'Adriano Negri',
+    title: 'Associate Producer | EA SPORTS FIFA',
+    info: 'Producer / designer on the highest-selling sports video game franchise in history: EA SPORTS FIFA.',
+    image: 'images/Negri.jpg',
+    class: 'show',
+    id: 'first',
+  },
```

Comment:
- [ ] Each speaker image is expected to look like the design, there should be a background behind each image.

| Original | Yours |
| ----- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/132952083-97a2c118-c4f6-4968-98fa-b810b645a21f.png' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/132952092-ff6b06dd-2690-474e-bef7-2b0538996c15.png' height='250px'> |

Reply:
We are allowed to use our own designs but still following the general format so I don't think there is a problem here.

Reply:
- [ ] Hi, let me share part of the guidelines for requirements. As you can see the style should be almost identical. 🐶 
![image](https://user-images.githubusercontent.com/20563252/132955007-754f1cc6-f930-4ba3-ab53-2e5410754d09.png)

Reply:
okay I'll add the background image